src/hotspot/share/runtime/objectMonitor.cpp

```diff
@@ -1071,11 +1068,6 @@ void ObjectMonitor::ReenterI(JavaThread* current, ObjectWaiter* currentNode) {
     OrderAccess::fence();
 
     // Keep a tally of the # of futile wakeups.
-    // Note that the counter is not protected by a lock or updated by atomics.
-    // That is by design - we trade "lossy" counters which are exposed to
-    // races during updates for a lower probe effect.
```

Comment:
I would suggest restoring the first three deleted lines as the counter update is still racy/lossy.

Reply:
Restored.